js/back.js

```diff
@@ -18,21 +18,27 @@
 
 "use strict";
 
-(function(global)
+(function (global) 
```

Comment:
Detail: Can you please fix this autocorrections(see also below) for the unrelated code please.